services/src/main/scala/cromwell/pipeline/service/GitLabProjectVersioning.scala

```diff
@@ -126,7 +128,25 @@ class GitLabProjectVersioning(httpClient: HttpClient, config: GitLabConfig)
         .recover { case e: Throwable => Left(VersioningException.HttpException(e.getMessage)) }
     }
 
-  override def getFiles(project: Project, path: Path)(implicit ec: ExecutionContext): AsyncResult[List[String]] = ???
+  override def getFiles(
```

Comment:
Looks like now this method can be re-used in `cromwell.pipeline.service.AggregationService#getFilesByProjectId`

Reply:
Done.